docs/meta/scope-of-pypdf.md

```diff
@@ -0,0 +1,62 @@
+# Scope of pypdf
+
+What features should pypdf have and which features will it never have?
+
+pypdf aims at making interactions with PDF documents simpler. Core tasks that
+pypdf can perform are:
+
+* Document manipulation: Splitting, merging, cropping, and transforming the pages of PDF files
+* Data Extraction: Extract text and metadata from PDF documents
+* Security: Decrypt / encrypt PDF documents
+
+Typical indicators that something should be done by pypdf:
+
+* The task needs in-depth knowledge of the PDF format
+* It currently requires a lot of code or even is impossible to do with pypdf
+* It's neither mentioned in "belongs in user code" nor in "out of scope"
+* It already is in the issue list with the [is-feature tag](https://github.com/py-pdf/pypdf/labels/is-feature).
```

Comment:
It might be interresting also to add ref to the roadmap and requested feature thread

Reply:
I've just linked the [moonshot extensions](https://github.com/py-pdf/pypdf/discussions/1181). Is this what you were thinking about?